Packs/AzureDevOps/ReleaseNotes/1_4_10.md

```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### Azure DevOps Modeling Rule
+
+- Updated the Azure DevOps Modeling Rule modeling rule to support ingestion via Azure Event hub.
```

Comment:
Updated the Azure DevOps Modeling Rule to support ingestion via Azure Event hub.  (we don't need to say modeling rule twice)